module_intros/density.CorrelationFunction-Grains.ipynb

```diff
@@ -0,0 +1,270 @@
+{
```

Comment:
<p>Can you use the same format string width specifiers for both the headers and the text? Especially because the spacing in the header makes it hard to see that there are two columns since it's a single space both between words and between columns.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/freud-examples/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>